qa/run-tests.xml

```diff
@@ -243,53 +244,52 @@
     </target>
     <target name="junit-jdbc-testsuite">
-        <junit-tests tests="jdbcresources01_oracle_thin_jndi"/>
-        <junit-tests tests="jdbcresources02_oracle_thin_jndi"/>
-        <junit-tests tests="jdbcresources01_pgsql_jndi"/>
-        <junit-tests tests="jdbcresources02_pgsql_jndi"/>
-        <junit-tests tests="jdbcresources01_mysql_jndi"/>
-        <junit-tests tests="jdbcresources02_mysql_jndi"/>
-        <junit-tests tests="jdbcresources01_ibmdb2_jndi"/>
-        <junit-tests tests="jdbcresources02_ibmdb2_jndi"/>
-        <!-- mssql needs different .jar for jdk 1.5 vs. 1.6 runtime -->
-        <!--
-        <condition property="mssql.driver" value="sqljdbc4.jar">
-          <equals arg1="${java.specification.version}" arg2="1.6"/>
-        </condition>
-        <property name="mssql.driver" value="sqljdbc.jar"/>
-        <junit-tests tests="jdbcresources01_mssqlserver_jndi"/>
-        <junit-tests tests="jdbcresources02_mssqlserver_jndi"/>
-        <junit-tests tests="jdbcresources01_sybase_jndi"/>
-        <junit-tests tests="jdbcresources02_sybase_jndi"/>
-        -->
-    </target>
-    <target name="junit-jdbc-ncl" depends="junit-jdbc-ncl-testsuite">
+    	<if>
+            <equals arg1="${profile}" arg2="postgres"/>
+                <then>
+                    <junit-tests tests="jdbcresources01_pgsql_jndi"/>
+                    <junit-tests tests="jdbcresources02_pgsql_jndi"/>
+                </then>
+            <elseif>
+                <equals arg1="${profile}" arg2="mysql"/>
+                <then>
+                    <junit-tests tests="jdbcresources01_mysql_jndi"/>
+                    <junit-tests tests="jdbcresources02_mysql_jndi"/>
+                </then>
+            </elseif>
+            <elseif>
+                <equals arg1="${profile}" arg2="oracle"/>
+                <then>
+                    <junit-tests tests="jdbcresources01_oracle_thin_jndi"/>
+                    <junit-tests tests="jdbcresources02_oracle_thin_jndi"/>
+                </then>
+            </elseif>
+            <elseif>
+                <equals arg1="${profile}" arg2="db2"/>
+                <then>
+                    <junit-tests tests="jdbcresources01_ibmdb2_jndi"/>
+                    <junit-tests tests="jdbcresources02_ibmdb2_jndi"/>
+                </then>
+            </elseif>
+    		<else>
+                <junit-tests tests="jdbcresources01_pgsql_jndi"/>
+                <junit-tests tests="jdbcresources02_pgsql_jndi"/>
+                <junit-tests tests="jdbcresources01_mysql_jndi"/>
+                <junit-tests tests="jdbcresources02_mysql_jndi"/>
+                <junit-tests tests="jdbcresources01_oracle_thin_jndi"/>
+                <junit-tests tests="jdbcresources02_oracle_thin_jndi"/>
+                <junit-tests tests="jdbcresources01_ibmdb2_jndi"/>
+                <junit-tests tests="jdbcresources02_ibmdb2_jndi"/>
+    		</else>
+        </if>
```

Comment:
+1

Reply:
Please can we describe this behaviour in https://github.com/jbosstm/narayana/blob/main/qa/README.txt? I think the default assumption would be to try to run against all databases as resource managers. I am not -1 on the change, but I feel it should be mentioned somewhere as it would not be my default assumption.

Reply:
I agree Tom! Thanks for your comment, I have just pushed a [change](https://github.com/jbosstm/narayana/pull/2186/files#diff-6b89d1c753d93fc494538be2fc5723a8c1999cdae858d496532f0da86e79a7a5) for the README

Reply:
Thanks!